test/functional/fixtures/regression/gh-628/test.js

```diff
@@ -0,0 +1,19 @@
+var expect = require('chai').expect;
+
+describe('[Regression](GH-628)', function () {
```

Comment:
It's better be moved to `Regression` sub-fixture of `t.click`. We create dedicated fixtures only for tests that involve different API.